<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-PHQCGIDH" ignored="0" name="Test" tests="4" failures="2" timestamp="2021-06-20T10:23:12 IST" time="43.51" errors="0">
  <testcase name="verifyLoginHeader" time="0.131" classname="com.qa.tests.LoginPageTest">
    <failure type="java.lang.AssertionError" message="expected [HOME] but found [Home]">
      <![CDATA[java.lang.AssertionError: expected [HOME] but found [Home]
at com.qa.tests.LoginPageTest.verifyLoginHeader(LoginPageTest.java:80)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- verifyLoginHeader -->
  <testcase name="LoginPageTitleTest" time="0.026" classname="com.qa.tests.LoginPageTest"/>
  <testcase name="LoginTest" time="6.241" classname="com.qa.tests.LoginPageTest">
    <failure type="java.lang.AssertionError" message="expected [CRMPRO] but found [Free CRM - CRM software for customer relationship management, sales, and support.]">
      <![CDATA[java.lang.AssertionError: expected [CRMPRO] but found [Free CRM - CRM software for customer relationship management, sales, and support.]
at com.qa.tests.LoginPageTest.LoginTest(LoginPageTest.java:73)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- LoginTest -->
  <testcase name="LoginTest" time="4.014" classname="com.qa.tests.LoginPageTest"/>
</testsuite> <!-- Test -->
